examples/howto/plot_record_extracellular_potentials.py

```diff
@@ -86,24 +86,17 @@
                         gridspec_kw={'height_ratios': [1, 3, 2]})
 
 # Then plot the aggregate dipole time series on its own axis
-dpl[trial_idx].copy().smooth(
-    window_len=window_len).plot(ax=axs[0], decim=decimate,
-                                show=False)
-
-voltage_offset = 50  # the spacing between individual traces
-voltage_scalebar = 200  # can be different from offset
-# we can assign each electrode a unique color using a linear colormap
-colors = plt.get_cmap('cividis', len(electrode_pos))
+dpl[trial_idx].smooth(window_len=window_len)
+dpl[trial_idx].plot(ax=axs[0], decim=decimate)
```

Comment:
it's a bit strange to have smoothing and decimation in separate functions because you should never be decimating without smoothing ... but I'll leave it like this for now